.poshchan/settings.json

```diff
@@ -1,7 +1,15 @@
 {
-    "build_targets": {},
+    "build_targets": {
+        "ci": "docker-PR",
```

Comment:
I don't think this will work.  I pretty sure this is why steve didn't add it.  I asked him to add the bot for reminders for now.

Reply:
The README in his repo shows how to use this, but I can revert it if you want.

Reply:
How do we set the value of `Project` here:  https://github.com/SteveL-MSFT/PoshChan-Bot/blob/f310dc44ec9ba6f6b4c5af433f31b78e7ff519e9/FunctionApp/AzDevOps-Rebuild/run.ps1#L5

Reply:
I think it is autoset

Reply:
I see... You cannot.  It's hard coded to the github project name, here:  https://github.com/SteveL-MSFT/PoshChan-Bot/blob/f310dc44ec9ba6f6b4c5af433f31b78e7ff519e9/FunctionApp/PoshChan-Bot/run.ps1#L53

You have to get @SteveL-MSFT 	 to make it configurable before it will work.

Let's remove it and merge the rest.

Reply:
Do you want to file an issue in steve's repo and link it here?

Reply:
Okay

Reply:
Maybe an issues for separate permissions for reminders too.

Reply:
Regarding the build targets, the most recent change to PoshChan generalizes this so hypothetically, if you put it into the settings file to match the names of the builds for this repo, it should just work